lisa/microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```
@@ -1338,6 +1424,701 @@ def _set_block_size_env_var(self, testcase: str) -> None:
+            ">/dev/null 2>&1 || "
+            "echo 0 | sudo tee /sys/devices/system/cpu/cpufreq/boost "
+            ">/dev/null 2>&1 || true",
+            shell=True,
+            sudo=True,
+        )
+
+        # C-states ≤ C1E (Intel-specific, best-effort)
+        self.node.execute(
+            "echo 1 | sudo tee /sys/module/intel_idle/parameters/max_cstate "
+            ">/dev/null 2>&1 || true",
+            shell=True,
+            sudo=True,
+        )
+
+        # THP → never (host)
+        self.node.execute(
+            "echo never | sudo tee /sys/kernel/mm/transparent_hugepage/enabled "
+            ">/dev/null 2>&1 || true",
+            shell=True,
+            sudo=True,
+        )
+
+        # irqbalance → ON
+        self.node.execute(
+            "sudo systemctl enable --now irqbalance 2>/dev/null || "
+            "sudo service irqbalance start 2>/dev/null || true",
+            shell=True,
+            sudo=True,
+        )
+
+        # Reserve hugepages (try 1GB first, fallback to 2MB)
+        hugepage_1g_path = (
+            f"/sys/devices/system/node/node{self._numa_node}/"
+            f"hugepages/hugepages-1048576kB/nr_hugepages"
+        )
+        hugepage_2m_path = (
+            f"/sys/devices/system/node/node{self._numa_node}/"
+            f"hugepages/hugepages-2048kB/nr_hugepages"
+        )
+
+        # Check if 1GB hugepages are available
+        result = self.node.execute(
+            f"[ -f {hugepage_1g_path} ]",
+            shell=True,
+        )
+
+        if result.exit_code == 0:
+            # Try 1GB hugepages (16GB total)
+            self.node.execute(
+                f"echo 16 | sudo tee {hugepage_1g_path} >/dev/null 2>&1 || true",
+                shell=True,
+                sudo=True,
+            )
+            # Verify allocation
+            verify = self.node.execute(
+                f"cat {hugepage_1g_path} 2>/dev/null || echo 0",
```

Remove all `2>/dev/null`, because it hides potential error details.